### ISSUE TITLE
would be nice to be able to download friends' submissions

### ISSUE THREAD

**Issue by iamlevis** [samirelanduk/reddit-save#14]
As it says in the title:  It would be nice to be able to download submissions made by a user's friends.

**Comment by samirelanduk** [comment]
Friends as in providing someone else's username and downloading their submissions? Could be extended to do that, but it's a bit off-mission. I am going to focus on this repo a bit in the next few weeks given everything that's happening, so I'll see how easy it would be to add.

**Comment by iamlevis** [comment]
I'm a half-decent programmer and have code that works.  But I don't know
how to do a pull-request in github; maybe you do.  Do I fork the project
first and then make my changes on that fork (rather than on a branch,
which is how it works at my work)?

**Comment by samirelanduk** [comment]
Yes that's right - fork, clone forked repo locally, make any commits, push commits to your forked repo on here, then open a pull request -  thanks a lot!
